webapp-django/crashstats/crashstats/models.py

```diff
@@ -1518,19 +1518,41 @@ class BugzillaAPI(SocorroCommon):
 
 class BugzillaBugInfo(BugzillaAPI):
 
-    def get(self, bugs, fields):
+    cache_seconds = 0
```

Comment:
Every model has a `cache_seconds`. So if you do `SomeClassInThisFile().get(**parameters)` it will cache the results for `self.cache_seconds`. That number is often not specified in the classes in this file. Because they all lazily inherit from the base class which sets the default (which I believe is 60). 

By setting this to `0` it means this model's internal `get()` doesn't read from or write to the cache. That's because I instead cache the individual bits inside the `.get()` method.